content/partials/pricing.html

```diff
@@ -0,0 +1,171 @@
+{#
+  It's not yet clear if the community plan features on the feature table help
+  explain the difference between community/commercial, or if it makes it less
+  obvious that community is only for open source projects.
```

Comment:
It would be nice if we could toggle them on & off, and default them off, but you can turn it on with a toggle.

Reply:
Mm, that could be a good option. Okay to leave this one for later? I was going to wait to see on this and maybe play more in another pass. I could see us lumping the Gold plan in the feature table too, but avoided making this table too confusing for now.

Reply:
Yea, definitely not a blocker.

Reply:
I put this one on my later list on #167. We can return to this and polish the UX on the community plans more, I think I'd like some time for this to simmer and see if we're confusing folks more or less with this listing.